syntax = "proto3";

package weather_notification;

// Weather service definition
service WeatherService {
  rpc GetWeather (WeatherRequest) returns (WeatherResponse);
}

message WeatherRequest {
  string location = 1;
}

message WeatherResponse {
  string location = 1;
  string condition = 2;
}

// Notification service definition
service NotificationService {
  rpc SendNotification (NotificationRequest) returns (NotificationResponse);
}

message NotificationRequest {
  string message = 1;
}

message NotificationResponse {
  string status = 1;
}
